hack/verify-rbac.sh

```
@@ -0,0 +1,123 @@
+
+ALL_VERBS="get list patch watch update delete create"
+EXCLUDE_DIRS="-not -path '*/vendor/*' -not -path '*/.git/*'"
+FOUND=0
+
+for verb in ${ALL_VERBS}; do
+  if is_allowed "${verb}"; then
+    continue
+  fi
+
+  pattern=$(verb_to_pattern "${verb}")
+  matches=$(find pkg -name '*.go' ${EXCLUDE_DIRS} \
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes in this variable will not be respected.

(SC2090)